app/component/icon/IconWithCaution.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import cx from 'classnames';
+
+const IconWithCaution = props => (
+  <span>
```

Comment:
Is this span required, we have it in the Icon, but only due to certain styles requiring that